arangod/Cluster/ClusterInfo.cpp

```diff
@@ -5726,6 +5726,31 @@ std::shared_ptr<std::vector<ServerID> const> ClusterInfo::getResponsibleServer(
   return std::make_shared<std::vector<ServerID>>();
 }
 
+ClusterInfo::ShardLeadership ClusterInfo::getShardLeadership(
+    ServerID server, ShardID shard) const {
```

Comment:
note: this line also needs adjustment after the corresponding header was changed.